recommendations/helpers.py

```diff
@@ -67,20 +85,42 @@ def make_readable_recommendation(recommended_films):
     for film in recommended_films:
         readable_film = {
             "id": film.id,
-            "name": film.name,
+            "name": f"{film.name} ({film.year})",
             "poster": film.poster,
-            "year": film.year,
+            "overview": film.overview,
             "language": "",
             "genre": "",
-            "director": "",
         }
 
+        if len(film.overview) > 100:
+            readable_film["overview"] = f"{film.overview[:100]}..."
+
         language_dict = dict(LANGUAGES)  # thanks chatgpt
         readable_film["language"] = language_dict.get(film.language)
 
-        readable_film["genres"] = (", ").join(film.genres)
-        readable_film["director"] = (", ").join(film.director)
+        readable_film["genres"] = (", ").join(film.genres[:3])
 
         readable_recommendation.append(readable_film)
 
     return readable_recommendation
+
+
+# if we have any requirements that django wouldn't normally catch
+def form_error_checking(field, selection):
+    if field == "Languages" and (len(selection) < 3 and "" not in selection):
+        return "You must pick at least three languages. If there are less than three, select 'No Preference'."
```

Comment:
This error seems strange, I think if there are less than 2 language options then the language form should be skipped BUT thats for another time (if at all)

Reply:
i disagree with this